core-bundle/src/Controller/ContentElement/AbstractContentElementController.php

```diff
@@ -59,5 +68,34 @@ protected function addSharedMaxAgeToResponse(Response $response, ContentModel $m
         $response->setSharedMaxAge(min($min));
     }
 
+    protected function isVisible(ContentModel $model, Request $request): bool
+    {
+        $isInvisible = $model->invisible || ($model->start && $model->start > time()) || ($model->stop && $model->stop <= time());
+
+        // The element is visible, so show it
+        if (!$isInvisible) {
+            return true;
+        }
+
+        return $this->isFrontendPreview($request);
+    }
+
+    protected function isFrontendPreview(Request $request): bool
```

Comment:
did you intentionally add this as a separate method because it could be used separately?

Reply:
I just followed the proposed solution mentioned here https://github.com/contao/contao/issues/2533#issuecomment-747503667

Reply:
Removed it in 0d18831.